lnrpc/lightning.proto

```diff
@@ -3506,7 +3506,30 @@ message PolicyUpdateRequest {
     // If true, min_htlc_msat is applied.
     bool min_htlc_msat_specified = 8;
 }
+
+message FailedUpdates {
+    enum Reason {
+        UNKNOWN = 0;
+        PENDING = 1;
+        INACTIVE = 2;
+        NOTFOUND = 3;
+        INVALID = 4;
+        INVALID_PARAMETER = 5;
+    }
```

Comment:
@carlaKC If you are not satisfied with the naming of these I'll change them.

Reply:
These look good! 

One comment I'd make is to move the enum out of the message and have it sit by itself, just makes things easier if we ever want to re-use the enum (if we move it later, it's a compile time break). 

Grpc has also been complaining about duplicate names recently (logging warnings etc), so I'd go with: 
```
enum UpdateFailure{
  UPDATE_FAILURE_UNKNOWN = 0;
  ...
  UPDATE_FAILURE_INVALID ...
```